index.html

```diff
@@ -0,0 +1,211 @@
+              </p>
+            </div>
+            <div class="event-item">
+              <img
+                class="event-image"
+                src="./images/icons/workshop.svg"
+                alt="Lecture Talk"
+              />
+              <h2>workshop</h2>
+              <p>
+                Learn fashion and designing from the experts.
+              </p>
+            </div>
+            <div class="event-item">
+              <img
+                class="event-image"
+                src="./images/icons/forum.svg"
+                alt="Lecture Talk"
+              />
+              <h2>Forum</h2>
+              <p>
+                Have the time to share your opinions and your thoughts with
+                expert designers and fashion moguls.
+              </p>
+            </div>
+            <div class="event-item">
+              <img
+                class="event-image"
+                src="./images/icons/exhibitions.svg"
+                alt="Lecture Talk"
+              />
+              <h2>Exhibitions</h2>
+              <p>
+                Showcase your brand to thousands of fashion Ethusiasts and get exlusive access to 
+                the main event. 
+              </p>
+            </div>
+          </div>
+          <button type="button" class="register-btn">
+            Register For the event
+          </button>
+          <a href="#" class="see-all-program">See the Whole Program</a>
+        </div>
+      </section>
+      <section id="trainers">
+        <div class="desktop-padding">
+          <h2 class="trainers-section-title the-underline">Designers</h2>
+          <div id="trainers-container"></div>
+          <button id="more-trainers" class="more-trainers">
+            More <img src="images/icons/down.svg" alt="" />
+          </button>
+        </div>
+      </section>
+      <section id="partners">
+        <h2 class="partners-section-title the-underline">Partners</h2>
+        <div class="partners-container">
+          <img
+            class="partner-image"
+            src="images/social//mtn.svg"
+            alt="mtn"
+          />
+          <img
+            class="partner-image"
+            src="images/social/Louis_Vuitton.svg"
+            alt="Louis_Vuitton"
+          />
+          <img
+            class="partner-image"
+            src="images/social/New_Balance.svg"
+            alt="New-Balance"
+          />
+          <img
+            class="partner-image"
+            src="images/social/toyota.svg"
+            alt="toyota"
+          />
+          <img
+            class="partner-image"
+            src="images/social//airtel.svg"
+            alt="airtel"
+          />
+          <img
+            class="partner-image"
+            src="images/social/shoprite.svg"
+            alt="shoprite"
+          />
+        </div>
+      </section>
+    </main>
+    <footer>
+      <p>2022 Lusaka fashion  week <br>all rights reserved.</p>
+    </footer>
```

Comment:
- [ ] Hi, this section requires that you have a logo aligned to the left and text aligned to the right currently you only have text. please try to match the design by implementing this feature.


Your Design | Reference Design
-|-
![image](https://user-images.githubusercontent.com/18730106/158031973-45d954c4-0f4f-44e3-a0d8-310e6e5c0024.png)|![image](https://user-images.githubusercontent.com/18730106/158031977-e7d49f5a-eb1e-46c4-8ff8-3a3412795e54.png)